src/fulltrade/FullTradeCard.js

```diff
@@ -57,11 +57,11 @@ export default class BaseTradeCard extends Component {
                     assetName={selectedSymbol}
                     onClosePanel={() => actions.removeTrade(index)}
                 />
-                {tick && <TradeChart
+                {(tick && Object.keys(tick).length > 0) && <TradeChart
```

Comment:
Yep, we will have transitioning states for the chart. Visually there should always be a chart on screen even if empty. In fact we might decide to return a <div>Loading...</div> from the chart component (or something similar).

Same thing for trading components, we should rarely or never just hide components (visually look bad). Alternative - disabled or loading states